### PR TITLE
Added relativity equation in Java

### PR DESCRIPTION
**Pull request by robertoduessmann**
Added relativity equation in Java.

For https://github.com/BaReinhard/Hacktoberfest-Mathematics/issues/3
refs https://github.com/robertoduessmann/physicalc

### CODE REVIEW

File: physics/relativity_equation/java/RelativityEquation.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/**
+ * Created by robertoduessmann on 03/10/17.
+ */
+public class RelativityEquation {
+
+	private static final double SPEED_LIGHT = 299792458;
+
+	public static double getEnergy(double mass) {
+		double energy = mass * Math.pow( SPEED_LIGHT, 2 );
```
Comment (stepfair) [inline physics/relativity_equation/java/RelativityEquation.java]: Do you need a local variable when we just return immediately?

Comment (mdew192837) [reply physics/relativity_equation/java/RelativityEquation.java]: Yeah, let's do that. It's clear this is energy. Maybe have a comment saying e = m*c^2

File: physics/relativity_equation/java/RelativityEquation.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+		return energy;
+	}
+
+	public static void main(String[] args) {
```
Comment (mdew192837) [inline physics/relativity_equation/java/RelativityEquation.java]: Remove this function

File: physics/relativity_equation/java/RelativityEquation.java (hunk 1 above)
Comment (robertoduessmann) [reply physics/relativity_equation/java/RelativityEquation.java]: Great, don't need! 
I created only to try be clearer, but isn't necessary... I will remove :+1:

File: physics/relativity_equation/java/RelativityEquation.java (hunk 2 above)
Comment (robertoduessmann) [reply physics/relativity_equation/java/RelativityEquation.java]: Nice, I've created only to test :)

**Comment by robertoduessmann** [comment]
> Rename to relativity_equation.java

Thanks for the feedbacks @mdew192837 and @stepfair

**Pull request merged**